gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/ManifestBasedDataset.java

```diff
@@ -117,6 +126,7 @@ public Iterator<FileSet<CopyEntity>> getFileSetIterator(FileSystem targetFs, Cop
         CommitStep step = new DeleteFileCommitStep(targetFs, toDelete, this.properties, Optional.<Path>absent());
         copyEntities.add(new PrePublishStep(datasetURN(), Maps.newHashMap(), step, 1));
       }
+      log.info(String.format("Workunits calculation took %s milliseconds to process %s files", System.currentTimeMillis() - startTime, numFiles));
```

Comment:
Should we measure this in nanoseconds? Typically that's what is used when instrumenting java code. Can convert units afterwards.

Reply:
I don't have a strong preference here, but calling nanoTime seems a little expansive. Also, in this case, where our computing time usually takes several seconds to several minutes, do we need that precise value?

Reply:
Oh I see, in regular distcp it takes a few milliseconds on avg which is why the granularity helps but manifest is different